modules/auxiliary/gather/search_email_collector.rb

```diff
@@ -42,6 +42,7 @@ def search_google(targetdom)
     print_status("Searching Google for email addresses from #{targetdom}")
     response = ""
     emails = []
+    header = { 'User-Agent' => "Mozilla/5.0 (compatible; Googlebot/2.1; +http://www.google.com/bot.html)"}
```

Comment:
Woops so this isn't a change (darn GitHub getting confused on changes on commits), but I still think its good to point out as technically speaking I don't think there is a reason this can't be randomized.